lib/mini_twitter_client.rb

```diff
@@ -0,0 +1,81 @@
+require 'faraday'
+require 'json'
+
+class Tweet
+  def initialize(name = 'name', email = 'email@email.com', message = 'message', id = nil)
```

Comment:
At the moment, I could use it this way:
```ruby
Tweet.new("Marek", "marek@marek.local", "Hey!", 5)
```

In this case, I think it is easy to mess up the order of the arguments (it is not clear that `name` should be first, etc.).

I think it would be more clear to use keyword arguments ([sample article about it](https://medium.com/@sologoubalex/keyword-parameters-in-ruby-e93d7a268151)):

```suggestion
  def initialize(name: 'name', email: 'email@email.com', message: 'message', id: nil)
```

They I would use it this way:

```ruby
Tweet.new(name: "Marek", email: "marek@marek.local", message: "Hey!", id: 5)
```

which I think is more readable.

Reply:
Also, I would require explicitly passing all of the arguments and not use default values in this case:

```suggestion
  def initialize(name:, email:, message:, id:)
```

I think it would be weird that I'm building a tweet `Tweet.new(id:5)` and it automatically has an email `email@email.com`, etc. I think I should get an error (with keywords arguments, the error would even say which argument is missing).